### PR TITLE
[68_Icons]

### LINKED ISSUE #68: Prepare basic set of icons
Should find free icons that can be used in project.
Should import them as a font.

### PR DESCRIPTION
**Pull request by karolkozlo**
[add] New WofIcon component added globally
[add] New icons.less style file with embeded font

How to test: Use WofIcon in template in App.vue. Pass class name of chosen icon (possible names in icons.less) via props.

closes #68

### CODE REVIEW

File: frontend/src/components/WofIcon.vue [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        },
+        size: {
+            type: Number,
+            default: 16
+        }
+    }
+}
+</script>
+
+<style lang="less">
```
Comment (kpagacz) [inline frontend/src/components/WofIcon.vue:23]: scoped or not?

File: frontend/src/components/WofIcon.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<template>
+    <div>
+        <i :class="icon" :style="`font-size: ${size}px`"></i>
```
Comment (kpagacz) [inline frontend/src/components/WofIcon.vue:3]: [This guide](https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended) recommends splitting lines on multiple attributes of an html element. Are we supposed to follow this rule or not?

File: frontend/src/components/icons.less [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+.wof-trash:before {
+    content: '\f1f8';
+}
+
+/* '' */
+.wof-user-delete:before {
+    content: '\f235';
+}
+
+/* '' */
```
Comment (kpagacz) [inline frontend/src/components/icons.less:178]: Probably?
```suggestion
/* '' */

```

File: frontend/src/components/WofIcon.vue (hunk 2 above)
Comment (kpagacz) [inline frontend/src/components/WofIcon.vue:3]: Also - in [all the examples I have seen](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1) the style bindings were done like this:
```suggestion
        <i :class="icon" :style="{ font-size: size + 'px' }"></i>
```
Is there a reason we shy away from it?

File: frontend/src/components/WofIcon.vue (hunk 1 above)
Comment (karolkozlo) [reply frontend/src/components/WofIcon.vue:23]: It doesn't matter here because this component will not have any other content.

**Review by kpagacz — APPROVED** [review]
I have some not so important styling questions. Also - how are we supposed to know what icons can go into this?

File: frontend/src/components/WofIcon.vue (hunk 2 above)
Comment (karolkozlo) [reply frontend/src/components/WofIcon.vue:3]: It's a good rule of thumb, but I think we use it for more severe cases

Comment (karolkozlo) [reply frontend/src/components/WofIcon.vue:3]: It's kinda a bad practice in JS

Comment (kpagacz) [reply frontend/src/components/WofIcon.vue:3]: Why?

**Pull request merged**